src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -1285,6 +1285,27 @@ Sum: 10, Count: 4")
         End Sub
 
         <Fact>
+        <WorkItem(18762, "https://github.com/dotnet/roslyn/issues/18762")>
+        Public Sub UnnamedTempShouldNotCrashPbdEncoding()
+
+            Dim verifier = CompileAndVerify(
+<compilation>
+    <file name="a.vb">
+Imports System
+Imports System.Threading.Tasks
+
+Module Module1
+    Private Async Function DoAllWorkAsync() As Task(Of (FistValue As String, SecondValue As String))
+        Return (Nothing, Nothing)
+    End Function
+End Module
+    </file>
+</compilation>,
+additionalRefs:={ValueTupleRef, SystemRuntimeFacadeRef}, useLatestFramework:=True, options:=TestOptions.DebugDll)
+
+        End Sub
+
+        <Fact>
```

Comment:
@AlekseyTs On the question of whether the function local should have a name in the debug info, one additional clue is that the following code gives and error:
```VB
Imports System
Imports System.Threading.Tasks

Module Module1
    Private Async Function DoAllWorkAsync() As Task(Of (FistValue As String, SecondValue As String))
        DoAllWorkAsync = (Nothing, Nothing) ' error BC36946: The implicit return variable of an Iterator or Async method cannot be accessed.
        Return (Nothing, Nothing)
    End Function
End Module
```